app/src/ui/branches/branch-list.tsx

```diff
@@ -162,6 +200,15 @@ export class BranchList extends React.Component<
     )
   }
 
+  private renderGroupHeader = (id: string) => {
+    const identifier = id as BranchGroupIdentifier
```

Comment:
:art: Rather than an implicit cast here, we could do an explicit lookup:

```ts
function parse(label: string): BranchGroupIdentifier | null {
  switch (label) {
    case 'default':
    case 'recent':
    case 'other':
      return label

    default:
      return null
  }
}

...

  private renderGroupHeader = (id: string) => {
    const identifier = parse(id)
    if (identifier == null) {
      return
    }

    return (
      <div className="branches-list-content filter-list-group-header">
        {this.getGroupLabel(identifier)}
      </div>
    )
  }

```